thoth/messaging/__init__.py

```diff
@@ -27,6 +27,11 @@
 from .unresolved_package import UnresolvedPackageMessage
 from .unrevsolved_package import UnrevsolvedPackageMessage
 from .message_factory import message_factory
+from .schedule_adviser import ScheduleAdviserMessage
+from .schedule_kebechet import ScheduleKebechetMessage
+from .schedule_package_extract import SchedulePackageExtract
+from .schedule_provenance_checker import ScheduleProvenanceCheckerMessage
+from .schedule_thamos import ScheduleThamosMessage
```

Comment:
Would it make sense to keep messages as they would describe events (instead of some sort of remote procedure call)? We can have different actions based on advise coming into the system (not just scheduling adviser).